projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-upload-media-v1-1-endpoint.php

```diff
@@ -105,6 +118,12 @@ public function callback( $path = '', $blog_id = 0 ) {
 			if ( count( $jetpack_media_files ) > 0 ) {
 				add_filter( 'upload_mimes', array( $this, 'allow_video_uploads' ) );
 
+				// get_space_used() checks blog upload directory storage,
+				// so filter it temporarily to return only video storage used.
+				if ( function_exists( 'videopress_filter_jetpack_get_space_used' ) ) {
+					add_filter( 'pre_get_space_used', 'videopress_filter_jetpack_get_space_used' );
+				}
```

Comment:
You missed the bit later in the function removing the added filter

```php
                // Remove get_space_used filter after upload.
                remove_filter( 'pre_get_space_used', 'videopress_filter_jetpack_get_space_used' );
```
Now that I think of it, probably that doesn't even need an `IS_WPCOM` guard. `remove_filter()` won't complain if the filter had never been added.

Reply:
Removed the previous function_exists check.